project/git.go

```diff
@@ -144,6 +155,14 @@ func (g gitProject) Update() error {
 	return nil
 }
 
+func recurseSubmodules(str string) bool {
+	switch strings.ToLower(str) {
+	case "0", "f", "false", "off", "no":
```

Comment:
I think we could make this simpler and allow only `off` (or `false`) 🤔 wdyt?

If we decide to support all these options, we should probably add tests for them as well.

Reply:
LOL, I agree. I basically copied [strconv.ParseBool](https://golang.org/src/strconv/atob.go?s=351:391#L1)'s implementation and added "no" and some case insensitivity, but we could pare this way down to not be so permissive. It also eats the error and assumes true if an invalid value is provided. That might not be what we want, but since disabling submodules is an edge case I didn't figure it was a big deal. I can iterate on this some more and add a test.

Reply:
I think we can change this to just parse bool with case insensitivity... so its simpler :)